src/NServiceBus.RabbitMQ/Sending/PoisonMessageForwarder.cs

```diff
@@ -1,37 +1,25 @@
 ﻿namespace NServiceBus.Transport.RabbitMQ
 {
-    using System;
     using System.Threading.Tasks;
     using global::RabbitMQ.Client.Events;
-    using Logging;
 
     class PoisonMessageForwarder
     {
-        static readonly ILog Logger = LogManager.GetLogger(typeof(PoisonMessageForwarder));
-
         readonly IChannelProvider channelProvider;
 
         public PoisonMessageForwarder(IChannelProvider channelProvider)
         {
             this.channelProvider = channelProvider;
         }
 
-        public Task ForwardPoisonMessageToErrorQueue(BasicDeliverEventArgs message, Exception ex, string errorQueue)
+        public Task ForwardPoisonMessageToErrorQueue(BasicDeliverEventArgs message, string errorQueue)
```

Comment:
The more I look at this the more I think we should drop this class and take a dep on `IChannelProvider` in `MessagePump`.